llvm/lib/Transforms/IPO/SampleProfile.cpp

```diff
@@ -234,6 +234,22 @@ static cl::opt<unsigned> ProfileICPRelativeHotnessSkip(
     cl::desc(
         "Skip relative hotness check for ICP up to given number of targets."));
 
+static cl::opt<unsigned> HotFuncCutoffForStalenessError(
+    "hot-func-cutoff-for-staleness-error", cl::Hidden, cl::init(999000),
+    cl::desc("A function is considered hot for staleness error check if its "
+             "total sample count is above the specified percentile(multiplied "
+             "by 10000, e.g. 995000 for 99.5 percentile)."));
```

Comment:
I don't think the explantion for `multiply by ...` is needed. We don't have that for cutoff in ProfileSummaryBuilder. But no strong opinion.